website/versioned_docs/version-0.57/navigatorios.md

```diff
@@ -201,7 +201,9 @@ NavigatorIOS uses `route` objects to identify child views, their props, and navi
 
 ### `barStyle`
 
-The style of the navigation bar. Supported values are 'default', 'black'. Use 'black' instead of setting `barTintColor` to black. This produces a navigation bar with the native iOS style with higher translucency.
+<!-- alex ignore retext-profanities -->
```

Comment:
Could you specify the exact rule you're ignoring? And in this case, you could wrap black in a code block as \`black\`